src/sections/Community/Handbook/community.js

```diff
@@ -3,29 +3,185 @@ import { Container, Row, Col } from "../../../reusecore/Layout";
+            </p>
+            <ul>
+              <li><img src={CGB} style={badgeStyle}/>Community</li>
+              <li><img src={LG} style={badgeStyle}/>Landscape</li>
+              <li><img src={LIH} style={badgeStyle}/>Image Hub</li>
+              <li><img src={MLL} style={badgeStyle}/>Meshery</li>
+              <li><img src={MOD} style={badgeStyle}/>Meshery Operator</li>
+              <li><img src={SDT} style={badgeStyle}/>Service Mesh Performance</li>
+              <li><img src={GNL} style={badgeStyle}/>GetNighthawk</li>
+              <li><img src={PL} style={badgeStyle}/>Patterns</li>
+            </ul>
+
+            <h2>Path to Leadership</h2>
+            <h3>What does leadership mean to the Layer5 community?</h3>
+            <p>
+              A leader is someone who can contribute to the Layer5 Community's growth by being accountable, participating in decision-making, and feeling responsible.
+            </p>
+            <h3>What does it take to be a leader?</h3>
+            <h4><strong>First, you have to be a MeshMate</strong></h4>
+            <p>
+              Roles/Responsibilities
+              <ul>
+                <li>Increasing awareness of the community to others</li>
+                <li>Helping newbies in the community get familiar with all of the projects</li>
+                <li>Mentoring members of the community</li>
+                <li>Facilitate newcomers call</li>
+              </ul>
+            </p>
+            <h4><strong>Maintainer</strong></h4>
+            <p>
+              Roles/Responsibilities
+              <ul>
+                <li>Send a reminder about meetings</li>
+                <li>Prepare meetings</li>
+                <li>Lead meetings</li>
+                <li>Make sure meeting minutes are kept</li>
+                <li>Facilitate the creation and advancement of metrics/software</li>
+                <li>Answer questions about the progress of Layer5 projects</li>
+                <li>Report on weekly community call progress on a project</li>
+                <li>Review issues on the repository</li>
+                <li>Review and merge pull requests</li>
+                <li>Regularly check the repository for stale content</li>
+                <li>Monitor issue tracker and pull requests</li>
+              </ul>
+            </p>
+            <p><strong>Checklist before becoming a Maintainer</strong></p>
+            <input type="checkbox" name="maintainer-checklist-1" value="Makes consistent contributions within the Layer5 community"/>Makes consistent contributions within the Layer5 community<br/>
+            <input type="checkbox" name="maintainer-checklist-2" value="Has the ability to commit directly to a project repository"/>Has the ability to commit directly to a project repository<br/>
```

Comment:
![R2](https://user-images.githubusercontent.com/85789734/133937139-3d5b69f0-f28b-4a22-9129-110e3622bb8f.jpg)
If you could, it'll be nice to change the checkbox color to something from our color scheme: maybe to caribbean green

Reply:
Yes, agree. Check the code of conduct page. We have one checkmark there.